.github/workflows/check-clippy.yaml

```diff
@@ -17,4 +17,4 @@ jobs:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         working-directory: .
         run: |
-          make clippy
+          cargo clippy -- -D warnings
```

Comment:
I think we can start with merging `-D warnings` and then add `-D all`. I think this kind of gradual transition will be more doable because then there I won't need to worry about new warning lints being added from other contributors while trying to fix the `-D all` lints

Reply:
that's fine with me. We do still need to make this change in the `clippy` makefile target, not here.